translations/de.json

```
@@ -0,0 +1,226 @@
+        "Contributors": "Unterstützer",  
+        "Craft": "anfertigen",  
+        "Cures": "heilt",  
+        "Default progress": "standard Vorgehen",  
+        "Delay": "verzögern",  
+        "Dmg": "Dmg",  
+        "Duration": "Dauer",  
+        "EFT Discord Bot": "EFT Discord Bot",  
+        "Enjoy": "viel Spaß",  
+        "Escape from Tarkov Status": "Escape from Tarkov Status",  
+        "Escort": "geleiten",  
+        "Examples": "Beispiele",  
+        "Fee": "Gebühr",  
+        "Flea Market": "Flohmarkt",  
+        "Flea Price (avg)": "Flohmarkt Preis (Durschnitt)",  
+        "Flea Price (low)": "Flohmarkt Preis (niedrigster)",  
+        "Flea Price (optimal)": "Flohmarkt Preis (optimiert)",  
+        "Found no results for \\"{{searchString}}\\"": "Keine Ergebnisse gefunden für: \\"{{searchString}}\\"",  
+        "for": "für",  
+        "Format HH:MM:SS": "Format HH:MM:SS",  
+        "Frag": "Splittergranate",  
+        "Get the full data from {{url}}": "Erhalte alle Informationen unter der URL: {{url}}",  
+        "Good Luck!": "Viel Glück!",  
+        "Health": "Gesundheit",  
+        "Help for /": "Hilfe für /",  
+        "Hideout": "Unterschlupf",  
+        "Hideout Management skill set to {{managementLevel}}.": "Unterschlupf Management Skill auf {{managementLevel}} gesetzt.",  
+        "I have been online for": "Ich bin online seit",  
+        "Intelligence Center": "Informationscenter",  
+        "Invalid token": "nicht gültiger Token",  
+        "Item Tier": "Gegenstandsabstufung",  
+        "kg": "kg",  
+        "Latest EFT Changes": "Letzte EFT Änderungen",  
+        "Last Updated": "Letzte Aktualisierung",  
+        "Level": "Level",  
+        "level": "level",  
+        "LL": "LL",  
+        "Loot Tiers": "Beutestuffen",  
+        "Loot tiers are divided primarily by the per-slot value of the item": "Beutestuffen werden primär durch ihren Ablagewert bestimmt.",  
+        "Map": "Karte",  
+        "Map made by {{author}}": "Erstellt von {{author}}",  
+        "minutes": "Minuten",  
+        "more": "mehr",  
+        "Name": "Name",  
+        "Need Help or Have Questions?": "Brauchst du Hilfe oder hast Fragen?",  
+        "never": "niemals",  
+        "Next update": "Nächste Aktualisierung",  
+        "No matching hideout station found.": "Keine passende Unterschlupf Ausbaumöglichkeit gefunden.",  
+        "No matching trader found.": "Keinen passender Händler gefunden.",  
+        "No prices available.": "Keine Preise verfügbar.",  
+        "Note: Progress synced via \[TarkovTracker\](<https://tarkovtracker.io/settings/>) will overwrite your hideout settings. \\nUse `/progress unlink` to stop syncing from TarkovTracker.": "Anmerkung: die synchronisierte Entwicklung über \[TarkovTracker\](<https://tarkovtracker.io/settings/>) wird deine Unterschlupf Einstellungen überschreiben. \\nVerwende den Befehl `/progress unlink` um dies zu stoppen",  
```

```suggestion
        "Note: Progress synced via [TarkovTracker](<https://tarkovtracker.io/settings/>) will overwrite your hideout settings. \nUse `/progress unlink` to stop syncing from TarkovTracker.": "Anmerkung: die synchronisierte Entwicklung über [TarkovTracker](<https://tarkovtracker.io/settings/>) wird deine Unterschlupf Einstellungen überschreiben. \nVerwende den Befehl `/progress unlink` um dies zu stoppen",  
```
Escapes not needed here too

---

Fixed in commit [c880c4b](https://github.com/the-hideout/stash/pull/125/commits/c880c4b5f3f62fc19379ca20d84a53b87adb653a)